src/Runtime/Builtins.hs

```diff
@@ -38,7 +39,8 @@ builtins = [
   ("inARow", \[Vi i, v, Vboard arr] -> return $ Vb $ inARow arr v i),
   ("next", \[Vs s] -> return $ if s == "X" then Vs "O" else Vs "X"),
   ("not", \[Vb b] -> return $ Vb (not b)),
-  ("or", \[Vb a, Vb b] -> return $ Vb (a || b))
+  ("or", \[Vb a, Vb b] -> return $ Vb (a || b)),
```

Comment:
do we want and/or to be builtins or in prelude?

Reply:
That's a good point. The current division is arbitrary. We could define everything other than the arithmetic operators, ==, and the board modification/access functions in the prelude. I won't do that for now though because the prelude is not working in our current front end.